adapter/src/main/java/com/twilio/kudu/sql/KuduQuery.java

```diff
@@ -115,6 +116,16 @@ public void register(RelOptPlanner planner) {
     if (CalciteSystemProperty.ENABLE_ENUMERABLE.value()) {
       KuduRules.ENUMERABLE_RULES.stream().forEach(r -> planner.addRule(r));
     }
+
+    // TODO remove if/when CALCITE-5420 is merged
+    planner.removeRule(MaterializedViewRules.AGGREGATE);
```

Comment:
disable the existing rule and enable the modified rule

Reply:
do we need to disable or can we just use a lower cost?

Reply:
We don't need to disable the old rule it just won't be used if the query has a filter. Since the new rule is doing the same work I thought its more efficient to disable the rule.